src/arena.rs

```diff
@@ -32,6 +33,10 @@ use const_fn_assert::{cfn_assert, cfn_assert_eq, cfn_assert_ne};
 /// If you provides `Arena` with invalid `LEM` or `BITARRAY_LEN`, then your
 /// code will panic at runtime:
 ///
+// We can wait for the `generic_const_exprs` feature to stabilize to simplify const generics.
+// This is the example: `buckets: Arcs<Arc<Bucket<T, BITARRAY_LEN, {BITARRAY_LEN * usize::BITS}>>>`
+// If possible, we can open an issue to track it first.
```

Comment:
I'd love that.

Having two separate const just complicated the API and make it more fragile.